source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,258 @@
+.. _c2c-filtered-sync:
+
+=============
+Filtered Sync
+=============
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. versionadded:: 1.1
+
+{+c2c-product-name+} provides continuous data synchronization or a 
+one-time data migration between two MongoDB clusters. You can use
+filtered sync to control the data that the :ref:`mongosync
+<c2c-mongosync>` utility transfers between the source and destination
+clusters.
+
+.. _c2c-configure-filter:
+
+Configure a Filter
+------------------
+
+.. important::
+
+   Once you start ``mongosync`` with a filter in place, the filter
+   cannot be modified. If you do need to create a new filter,
+   see: :ref:`c2c-change-filter`.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Identify Databases and Collections.
+
+      Identify the databases and collections that you want to sync to
+      the destination cluster. When you add a set of databases to the
+      filter, you also exclude any other databases in the cluster.
+      
+      If specify a collection in your filter, you also exclude any other
+      collections that are in the same database.
+
+   .. step:: Create a Filter.
+
+      The ``includeNamespaces`` parameter specifies an optional filter
+      that you can pass to the :ref:`/start <c2c-api-start>` API.
+      
+      If you don't specify a filter, ``mongosync`` does a full cluster
+      sync.
+
+      The filter syntax is: 
+
+      .. include:: /includes/api/facts/includeNamespaces-syntax.rst
+
+      Create an entry in the ``includeNamespaces`` array for each
+      database that you identified in step 1. Use the ``"database"``
+      field to specify the database name.
+      
+      If you want to filter on collections within a database, add those
+      collections to the list in the ``"collections"`` field for that
+      database entry.
+
+   .. step:: Use the Filter.
+
+      To use the filter, attach the filter document when you make the
+      :ref:`/start <c2c-api-start>` API call to begin syncing.
+
+      .. literalinclude:: /includes/api/requests/start-filtered.sh
+         :language: shell
+
+For an example configuration, see: :ref:`c2c-example-start-with-filter`.
+
+.. _c2c-change-filter:
+
+Replace an Existing Filter
+--------------------------
+
+You cannot update an existing filter. You must stop the ongoing sync
+process, prepare the destination cluster, and restart ``mongosync`` with
+a new filter. 
+
+When ``mongosync`` ran your original filter, it created databases with
+your data, "user databases", and the
```

Comment:
Can you put "user databases" in parenthesis. Otherwise it looks like your data and user dbs are different elements in the comma separated list

Reply:
Done